src/routes/following.route.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response } from "express";
+
+import { isAuthorized } from "../middlewares/isAuthorized.middlewares";
+import {
+  getFollowing,
+  addFollowing,
+  deleteFollowing,
+} from "../services/following.service";
+
+const followingRouter = express.Router();
+
+followingRouter
+  .route("/")
+  .get(isAuthorized, async (req: Request, res: Response) => {
+    const result = await getFollowing(req.query);
```

Comment:
```suggestion
  .get(isAuthorized, async (req: Request, res: Response) => {
    const result = await getFollowing(new PaginationParameters(req));
```
 tweet router for example